reel/migrations/0001_initial.py

```diff
@@ -1,96 +0,0 @@
-# Generated by Django 4.0.6 on 2022-07-29 05:36
```

Comment:
It is best not to delete migration files as it can leave database in inconsistent state. Especially during production.